db/migrate/201509151000_create_qualifying_comments_table.rb

```diff
@@ -0,0 +1,12 @@
+class CreateQualifyingCommentsTable < ActiveRecord::Migration
+  def change
+  	create_table :qualifyings do |t|
```

Comment:
If you had a class called `Qualifying` then it would look for the plural for the tables `Qualifyings`, but if you have a class called `QualifyingComment`, then it would look for `qualifying_comments`.

I think before we can find a great name for this I need to know how we're going to be using it.